### ISSUE TITLE
Expose the wabt interpreter API

### ISSUE THREAD

**Issue by axic** [pepyakin/wabt-rs#44]

**Comment by yurydelendik** [comment]
The alternative is to use [wasm-c-api](https://github.com/WebAssembly/wasm-c-api/) -- there is https://github.com/WebAssembly/wabt/issues/1158 filed for that. It might be a good idea to develop generic wasm-c-api wrapper, so the users will be agnostic to an engine implementation (wabt, v8, wasmtime).

**Comment by axic** [comment]
@yurydelendik yes, wasm-c-api would be nice, but still waiting for VMs to adopt it.

**Comment by yurydelendik** [comment]
> wasm-c-api would be nice, but still waiting for VMs to adopt it.

@axic The wasm-c-api is being developed on top of v8. I'm currently working on its implementation in wasmtime.

**Comment by sbc100** [comment]
I think we can close this in a favor of #1158
